src/nvim/buffer_defs.h

```diff
@@ -603,6 +603,7 @@ struct file_buffer {
   char_u *b_p_bt;               ///< 'buftype'
   int b_has_qf_entry;           ///< quickfix exists for buffer
   int b_p_bl;                   ///< 'buflisted'
+  long b_p_channel;             ///< 'bomb'
```

Comment:
Copy paste error: `bomb`? Comment should be `channel`.

Reply:
Do we need a bigger type for the channel number or is `INT_MAX` an upper limit that we don't reach?

Reply:
#6917 will make this limit explicit.